viewing_party/party.py

```diff
@@ -1,23 +1,173 @@
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    if title and genre and rating:
+        movie_dict = {}
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+    else:
+        return None
+    
+    return movie_dict
+
+
+def add_to_watched(user_data, movie):
+

```

Comment:
Finding a balance of whitespace inside of a function can be hard, and is often subjective. That said, for a very short function like this with a single statement and return, I would recommend removing the new lines on line 18 & 20 since this function is so small that everything is tightly related. This applies to the `add_to_watchlist` function below as well.

Reply:
This makes sense! I'll try to remember this and change my habits regarding whitespace. Totally makes sense to use whitespace as an indicator of how tightly related lines of code are.